src/main/java/nustracker/ui/HelpWindow.java

```diff
@@ -36,7 +36,8 @@
 
     public static final String EDIT_COMMAND = "edit";
     public static final String EDIT_COMMAND_EXAMPLE = " " + EDIT_COMMAND
-            + " [id/ID_OF_STUDENT_TO_EDIT] [n/NAME] [m/MAJOR] [id/NEW_STUDENT_ID] [y/YEAR] [p/PHONE] [e/EMAIL]";
+            + " [id/ID_OF_STUDENT_TO_EDIT] [n/NEW_NAME] [m/NEW_MAJOR]"
```

Comment:
```suggestion
            + " id/ID_OF_STUDENT_TO_EDIT [n/NEW_NAME] [m/NEW_MAJOR]"
```
The first id for edit command is compulsory